### PR TITLE
Implement Risk Management Agent #6

### PR DESCRIPTION
**Pull request by balbanyan**
Risk report generator using RiskAnalyzer class, including calculations for beta, systematic, and unsystematic risk

### CODE REVIEW

**Comment by balbanyan** [comment]
Changes summary:

-Fixed an error where missing data was leading to misaligned dates (mismatch in lengths of datasets) before calculating the covariance. This error occurred when using tickers from TASI due to incomplete data.  dropna() function was not enough to resolve this issue, therefore I added an align() function to ensure both stock and market returns were aligned to the same dates.

-Added a time period to the risk assessment report generator

File: src/classes.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
+from imports import *
```
Comment (SaudAltamimi) [inline src/classes.py]: I prefer to see the imports in the same file. It's easier to track.

File: src/classes.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+#Dummy variables
```
Comment (SaudAltamimi) [inline src/classes.py]: For testing the calss using dummy variables. I like to have them seperated in either unit tests or notebooks.

**Review by SaudAltamimi — CHANGES_REQUESTED** [review]
Overall it looks great. I just need three changes:

- [ ] Creating a notebooks example. In `notebooks/` folder outside `src/` folder
- [ ] remove dummy tickers from the same class file.
- [ ] Keep imports in the same python file

File: src/classes.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        self.lookback_days = lookback_days
+        self.stock_returns = None
+        self.market_returns = None
+        self.start_date = None
+        self.end_date = None
+    
+    # Fetch historical prices and compute daily returns
+    def get_returns(self, ticker):
+        self.end_date = datetime.now().date()
+        self.start_date = self.end_date - timedelta(days = self.lookback_days)
```
Comment (SaudAltamimi) [inline src/classes.py]: Can you add the lookback period variable into the class variables. We might need to adjust it for expirmentation.

Comment (SaudAltamimi) [reply src/classes.py]: It's better for now to keep it `lookback_years`

**Review by SaudAltamimi — CHANGES_REQUESTED** [review]
Can you just add the code inside a specific folder `src/value_mpt/risk`

**Review by SaudAltamimi — APPROVED** [review]

**Pull request merged**